crypto/bls.go

```diff
@@ -472,15 +441,15 @@ type pubKeyBLSBLS12381 struct {
 	// sure the comparison is performed after an instance is created.
 	//
 	// public key G2 point
```

Comment:
> the type name has changed to E2 to match the new BLST library's type name

`E2` and `G2` naming are not related to the use of Relic or BLST. `E2` is a set of points and `G2` is a subset of `E2`. The type has been renamed to `E2` because technically it can store points from the large set. However this specific structure guarantees that a point is always on the smaller, that's why the documentation only talks about `G2`.